src/test/jtx/Env.h

```diff
@@ -785,6 +792,9 @@ Env::rpc(std::string const& cmd, Args&&... args)
         std::forward<Args>(args)...);
 }
 
+extern std::string blob257;
```

Comment:
please add comment what these things are (e.g. blobs equivalent to N many bytes)